src/components/ListPage/ListPagesHelper.ts

```diff
@@ -7,7 +7,7 @@ import * as MessageCenter from '../../utils/MessageCenter';
 
 export namespace ListPagesHelper {
   export const perPageOptions: number[] = [5, 10, 15];
-  const defaultDuration = 600;
+  const defaultDuration = config().toolbar.defaultRateInterval;
```

Comment:
In config there's already something called "defaultDuration" but which is set to 1 minute.
This is why I think some default values, such as this one, should not be kept in global config be should be just set per page.

Reply:
The problem is that the graph use this value in the interval duration and toolbar and I don't know if this impact in something else, maybe we can ask to saturn team

Reply:
I know this is why I think it's better to *not* have it as a global config

Reply:
ok I'll keep the configuration in the type so

Reply:
I agree that the config needs some attention.  'Toolbar' should contain only setting global to every toolbar (is there anything actually global to every toolbar?).  Otherwise setting should be under qualified names.  Having said that, we should settle on some common naming for things like pollInterval/refreshInterval.  intervalDuration/duration  (what even is an intervalDuration :) ), etc.